dl4j-examples/tutorials/Dockerfile

```diff
@@ -88,15 +88,13 @@ RUN cd /usr/src/zeppelin \
  && rm -rf /usr/src/zeppelin \
  && rm -rf /root/.m2 \
  && rm -rf /root/.npm \
- && rm -rf /tmp/*
+ && find /tmp -maxdepth 1 -not -name 'apache-maven-3.5.0' -not -name "." -exec rm -rf {} \; \
+ # Removing extra interpreter folders
+  && find $ZEPPELIN_HOME/interpreter -maxdepth 1 -type d -not -name 'spark' -not -name 'md' -not -name "." -exec rm -rf {} \;
```

Comment:
As determined, this does not affect Zeppelin's runtime config and works as hoped.